go.mod

```diff
@@ -33,10 +33,13 @@ require (
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/u-root/uio v0.0.0-20240224005618-d2acac8f3701 // indirect
 	github.com/xrash/smetrics v0.0.0-20240521201337-686a1a2994c1 // indirect
-	golang.org/x/crypto v0.41.0 // indirect
-	golang.org/x/mod v0.27.0 // indirect
-	golang.org/x/net v0.42.0 // indirect
+	golang.org/x/crypto v0.42.0 // indirect
+	golang.org/x/mod v0.28.0 // indirect
+	golang.org/x/net v0.43.0 // indirect
 	golang.org/x/time v0.5.0 // indirect
-	golang.org/x/tools v0.35.0 // indirect
+	golang.org/x/tools v0.36.0 // indirect
 	gvisor.dev/gvisor v0.0.0-20240916094835-a174eb65023f // indirect
 )
+
+// Point to upstream main until the next release to get this fix: https://github.com/containers/gvisor-tap-vsock/commit/c4615cdb1462cb4c9ea5ab770ea11ad03c472c26
+replace github.com/containers/gvisor-tap-vsock => github.com/containers/gvisor-tap-vsock v0.8.8-0.20250909141233-c4615cdb1462
```

Comment:
Why replace?

Reply:
because it is a temporary replace until next tag